core/src/main/res/values/styles.xml

```diff
@@ -32,7 +32,7 @@
     <item name="android:background">@color/black</item>
   </style>
 
-  <style name="ThemeOverlay.KiwixTheme.Launcher" parent="">
+  <style name="KiwixTheme.Launcher">
```

Comment:
This has substantially changed, Why is this a child of KiwixTheme now?

Reply:
Themeoverlay caused the splash screen to be completely white with no icon. Using only KiwixTheme.Launcher works.

Reply:
However, the launch_screen icon does not automagically change to the night mode icon if dark mode is enabled.